cronJobs.js

```diff
@@ -0,0 +1,51 @@
+var CronJob = require('cron').CronJob;
+var moment = require('moment');
+
+module.exports = async function(db) {
+  new CronJob('0 9 * * *', async function() { // replace with * * * * * to run every minute for testing
+    const now = new Date();
+    const weekFromNow = new Date();
+    weekFromNow.setDate(now.getDate() + 7);
+    const requirementsAboutToExpire = await db.get('projectRequirementsCollection').find({
+      $and: [
```

Comment:
Can you make a method to get this query?

Reply:
y

Reply:
We are going to let this out for now